rfcs/0006-runtime-references-interface.md

```diff
@@ -44,6 +45,11 @@ its builtin hash scanning and just use the file. Both files are
 newline-separated lists of store paths. Parse errors in the
 `runtime-references` file, or non-existent store paths, fail the build.
 
+To avoid duplicating the default reference scanning behavior in Nix,
+we could also make a tiny tool part of Nix itself that wraps the
+`scanForReferences` API, and then expose that in the `<nix>` namespace
+so tooling can get the full path to that tool.
```

Comment:
If we move it out of Nix, we'd still need to inject the paths to scan for somehow, right? Would we just unconditionally pass those in as an env var? Or search for anything that looks like /nix/store/foo? I kind of like that you can't hardcode a random store path right now though.